modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_ar.properties

```diff
@@ -5511,6 +5511,7 @@ deleting-this-event-will-cancel-the-meeting-with-your-guests-would-you-like-to-d
 deletion-conflicts-with-a-modification=الحذف يتعارض مع التعديل.
 deletion-modification-conflict=حذف تعارض التعديلات
 deletion-not-allowed=الحذف غير مسموح به
+deletion-not-allowed-because-a-relationship-field-is-being-used-in-the-object-definition-x-to-proceed-introduce-a-new-custom-field-in-x-or-delete-relationship-x=Deletion not allowed because a relationship field is being used in the object definition {0}. To proceed, introduce a new custom field in {1} or delete relationship {2}. (Automatic Copy)
```

Comment:
I believe that you forgot re-run the `buildLang` after change the message.

Reply:
Indeed, sorry about that, I've just fixed it.